app/stores/user-products.ts

```diff
@@ -0,0 +1,86 @@
+export const useUserProductsStore = defineStore('bcreg-user-products-store', () => {
+  const accountStore = useConnectAccountStore()
+  const ldStore = useConnectLaunchdarklyStore()
+  const productInfo = useProductInfo()
+  // const { $authApi, $keycloak } = useNuxtApp()
+  const { $keycloak } = useNuxtApp()
+  const authUrl = useRuntimeConfig().public.authApiURL
+
+  const userProducts = ref<Product[]>([])
+  const loading = ref<boolean>(false)
+
+  async function getUserProducts () {
+    try {
+      loading.value = true
+      userProducts.value = []
+      // auth api not returning an array for some reason
+      // const response = await $authApi(`/orgs/${accountStore.currentAccount.id}/products?include_hidden=true`)
```

Comment:
$fetch util not returning array correctly but native fetch does, not sure whats going on there